Marketplace/CreateUIDefinition.json

```diff
@@ -0,0 +1,258 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+    "handler": "Microsoft.Azure.CreateUIDef",
+    "version": "0.1.2-preview",
+    "parameters": {
```

Comment:
Add the config at the beginning that allows for deploying to existing resource groups: https://github.com/InRule/DynamicsCRM/pull/323/files?diff=split&w=1